### ISSUE TITLE
Casteling should move King and Rook

### ISSUE THREAD

**Issue by theneongrey** [theneongrey/chess#11]
Used castleling casteling to queenside on white, but only king moved, rook stayed on initial place and was clickable there after the next round.
Game:

1.b2-b4 c7-c5
2.Nb1-c3 d7-d5
3.Bc1-a3 b7-b5
4.Qd1-b1

Expected left white rook on A3 but was on A0

**Comment by theneongrey** [comment]
No casteling was performed... just the queen was moved...
When performing a real queen side casteling though, the rook moves to the wrong position:
1.d2-d4 d7-d6
2.Bc1-f4 e7-e6
3.Qd1-d3 c7-c6
4.Nb1-a3 b7-b6
5.O-O-O

Result:
```
♜ | ♞ | ♝ | ♛ | ♚ | ♝ | ♞ | ♜
-- | -- | -- | -- | -- | -- | -- | --
♟ |   |   |   |   | ♟ | ♟ | ♟
  | ♟ | ♟ | ♟ | ♟ |   |   |  
  |   |   |   |   |   |   |  
  |   |   | ♙ |   | ♗ |   |  
♘ |   |   | ♕ |   |   |   |  
♙ | ♙ | ♙ |   | ♙ | ♙ | ♙ | ♙
  |   | ♔ |   | ♖ | ♗ | ♘ | ♖
```
But rook should be on A4

**Comment by theneongrey** [comment]
Was fixed in commit 3219af84ef0f06738cc1d71aceb4e02003324852
